dittybopper/deploy.sh

```diff
@@ -120,7 +121,7 @@ fi
 echo ""
 echo -e "\033[32mGetting environment vars...\033[0m"
 export PROMETHEUS_URL="https://$($k8s_cmd get routes -n openshift-monitoring prometheus-k8s -o jsonpath="{.spec.host}")"
-export PROMETHEUS_BEARER=$($k8s_cmd sa get-token -n openshift-monitoring prometheus-k8s || $k8s_cmd sa new-token -n openshift-monitoring prometheus-k8s)
+export PROMETHEUS_BEARER=$($k8s_cmd create token -n openshift-monitoring prometheus-k8s --duration=6h || $k8s_cmd sa get-token -n openshift-monitoring prometheus-k8s || $k8s_cmd sa new-token -n openshift-monitoring prometheus-k8s)
```

Comment:
Curious about the duration here?

Reply:
using the same timing as e2e-benchmark [here](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/dfa9d314b3ea05217c5debd8518f590ec4411f0d/workloads/kube-burner/common.sh#L12). Can increase it to 12h

Reply:
Can update with no timeout, but this is the standard we have used in other repos. Thoughts?

Reply:
I have clusters that stay up longer than 6 hours. I still vote to remove it. Raised a separate PR.